rdmo/locale/de/LC_MESSAGES/django.po

```diff
@@ -4337,6 +4351,139 @@ msgstr "Datei"
 msgid "Set"
 msgstr "Set"
 
+#~ msgid "Autocomplete"
```

Comment:
these are all commented `#` right? Do you want to keep them in?

Reply:
Hmm, I thing we should keep them to improve automatic translations, but we never decided anything on this.